bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetThermoregulationHandler.java

```diff
@@ -0,0 +1,375 @@
+                            getThing().getUID());
+                    return;
+                }
+            }
+        } else {
+            logger.warn("handleSetFanSpeedCommand() Unsupported command {} for thing {}", command, getThing().getUID());
+        }
+    }
+
+    private void handleSetpointCommand(Command command) {
+        logger.debug("handleSetpointCommand() (command={})", command);
+        if (command instanceof QuantityType || command instanceof DecimalType) {
+            Where w = deviceWhere;
+            if (w != null) {
+                BigDecimal value = BigDecimal.ZERO;
+                if (command instanceof QuantityType) {
+                    Unit<Temperature> unit = CELSIUS;
+                    QuantityType<Temperature> quantity = commandToQuantityType(command, unit);
+                    value = quantity.toBigDecimal();
+                } else {
+                    value = ((DecimalType) command).toBigDecimal();
+                }
+                try {
+                    send(Thermoregulation.requestWriteSetpointTemperature(w.value(), value.floatValue(),
+                            currentFunction));
+                } catch (MalformedFrameException | OWNException e) {
+                    logger.warn("handleSetpointCommand() {}", e.getMessage());
+                }
+            }
+        } else {
+            logger.warn("handleSetpointCommand() Unsupported command {} for thing {}", command, getThing().getUID());
+        }
+    }
+
+    private void handleMode(Command command) {
+        if (command instanceof StringType) {
+            Where w = deviceWhere;
+            if (w != null) {
+                try {
+                    OPERATION_MODE mode = OPERATION_MODE.valueOf(((StringType) command).toString());
+                    if (mode == OPERATION_MODE.MANUAL) {
+                        logger.debug("handleMode() mode={} function={} setPointTemp={}°", mode.toString(),
+                                currentFunction.toString(), currentSetPointTemp);
+                    } else {
+                        logger.debug("handleMode() mode={} function={}", mode.toString(), currentFunction.toString());
+                    }
+                    send(Thermoregulation.requestWriteMode(w.value(), mode, currentFunction, currentSetPointTemp));
+                } catch (OWNException e) {
+                    logger.warn("handleMode() {}", e.getMessage());
+                } catch (IllegalArgumentException e) {
+                    logger.warn("handleMode() Unsupported command {} for thing {}", command, getThing().getUID());
+                    return;
+                }
+            }
+
+        } else {
+            logger.warn("handleMode() Unsupported command {} for thing {}", command, getThing().getUID());
+        }
+    }
+
+    private void handleFunction(Command command) {
+        logger.debug("handleFunction() (command={})", command);
```

Comment:
Can this message be replaced by using the debugger? See point 4 https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
yes, I will remove it.